modules/integration/src/test/scala/scala/cli/integration/BspTestDefinitions.scala

```diff
@@ -273,7 +273,8 @@ abstract class BspTestDefinitions(val scalaVersionOpt: Option[String])
 
     withBsp(inputs, Seq(".")) { (root, _, remoteServer) =>
       async {
-        val buildTargetsResp = await(remoteServer.workspaceBuildTargets().asScala)
+        val buildTargetsResp =
+          Await.result(remoteServer.workspaceBuildTargets().asScala, 60.seconds)
```

Comment:
Did you change this to make it timeout earlier? Aren't the timeouts in `withBsp` enough?

If you really want to make just this future timeout earlier, [this](https://stackoverflow.com/a/45272591/3714539) can be a clean way of doing that.